mongo.py

```diff
@@ -0,0 +1,63 @@
+import pymongo
+from bson import ObjectId
+
+class Model(dict):
+    __getattr__ = dict.get
+    __delattr__ = dict.__delitem__
+    __setattr__ = dict.__setitem__
+
+    def save(self):
+        if not self._id:
+            self.collection.insert(self)
+        else:
+            self.collection.update(
+                {"_id": ObjectId(self._id)}, self)
+        self._id = str(self._id)
+
+    def reload(self):
+        if self._id:
+            result = self.collection.find_one({"_id": ObjectId(self._id)})
+            if result:
+                self.update(result)
+                self._id = str(self._id)
+                return True
+        return False
+    def remove(self):
+        if self._id:
+            resp = self.collection.remove({"_id": ObjectId(self._id)})
+            self.clear()
+            return resp
+
+class Listing(Model):
+    db_client = pymongo.MongoClient('localhost', 27017)
+    collection = db_client["listings"]["listings_list"]
```

Comment:
Same idea as MongoClient, the database now has different names, merge line from Master.

Reply:
Look at the comment for the back_end.py (2 above) !